tests/unit/CorrectBusinessContactInfo.spec.ts

```diff
@@ -0,0 +1,129 @@
+// Libraries
```

Comment:
Probably need to rename this, since you deleted the actual component by this name.

Reply:
Shoot, just needs to be renamed to `BusinessContactInfo.spec`

Reply:
> Shoot, just needs to be renamed to `BusinessContactInfo.spec`

Done in next commit!